oaknational/Oak-Web-Application
# PUPIL-327 replace MaxWidth, Grid and GridArea with oak-components

## Description

Music year: 2021

In About us pages:
- replace`MaxWidth` with `OakMaxWidth`
- replace `Grid` and `GridArea` with `OakGrid` and `OakGridArea`
- replace SharedComponents with oak-components in `landingPageSlug`
- replace SharedComponents with oak-components in `policyPageSlug`
- replace `LI` with `OakLI` in Partners page

## Issue(s)

Fixes #PUPIL-327

## How to test

1. Go to https://deploy-preview-2204--oak-web-application.netlify.thenational.academy
2. Go to About us pages
3. Check that nothing looks odd (only spacing could potentially got affected with this PR)
